scripts/data/preferences/README.md

```diff
@@ -0,0 +1,57 @@
+# Current preference datasets
```

Comment:
Should we move this to `docs/datasets`? Eventually, we should move the `scripts/readme.md` there somehow, too.

Reply:
oh we have docs!